drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-marshaller/src/main/resources/org/drools/workbench/scenariosimulation/kogito/marshaller/js/SCESIMMainJs.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @type {{marshall: MainJs.marshall, unmarshall: MainJs.unmarshall}}
+ */
+SCESIMMainJs = {
```

Comment:
isn't this automatically generated? do we need to track it?

Reply:
@jomarko That class is not automatically generated